crimsobot/bot.py

```diff
@@ -1,20 +1,20 @@
-import asyncio
 import logging
 import random
-from typing import Any, List, Union
+from typing import Any, List, Union, Mapping
 
 import discord
 from discord.ext import commands
 
 from config import ADMIN_USER_IDS, BANNED_GUILD_IDS, DM_LOG_CHANNEL_ID, LEARNER_CHANNEL_IDS, LEARNER_USER_IDS
 from crimsobot import db
 from crimsobot.models.ban import Ban
-from crimsobot.utils import checks, markov as m, tools as c
+from crimsobot.utils import markov as m, tools as c
 
 
 class CrimsoBOT(commands.Bot):
-    def __init__(self, **kwargs: Any) -> None:
+    def __init__(self, **kwargs: Mapping) -> None:
```

Comment:
This needs to remain `Any`, as this typing is for the _values_ of the kwargs.
See https://stackoverflow.com/questions/37031928/type-annotations-for-args-and-kwargs.

Reply:
Fixed in commit.